gio/Gir.toml

```diff
@@ -327,13 +327,26 @@ generate_builder = true
 name = "Gio.Cancellable"
 status = "generate"
 concurrency = "send+sync"
+manual_traits = ["CancellableExtManual"]
+    [[object.function]]
+    name = "connect"
+    # Needs manual bindings for closures
+    ignore = true
```

Comment:
This should get `manual = true` and `rename = `connect_cancelled`

Reply:
The only reason for that is the docs right? I ended up rewriting the docs to get rid of the C-isms, so I think `ignore` is appropriate here so it doesn't end up getting the C docs?

Reply:
Yes, that's OK then :)